plugins/recruitee/src/dataSources.ts

```diff
@@ -0,0 +1,311 @@
+    ]
+)
+
+const candidatesSchema = v.array(v.object({ candidates: v.array(CandidateSchema) }))
+const candidatesDataSource = createDataSource(
+    {
+        name: "Candidates",
+        fetch: async (token: string, companyId: string) => {
+            const url = `https://api.recruitee.com/c/${companyId}/candidates`
+            const data = v.safeParse(candidatesSchema, await fetchRecruiteeData(url, token))
+            if (!data.success) {
+                console.log("Error parsing Recruitee data:", data.issues)
+                throw new Error("Error parsing Recruitee data")
+            }
+            return data.output.flatMap(page => page.candidates)
+        },
+    },
+    [
+        { id: "id", name: "ID", type: "string", canBeUsedAsSlug: true },
+        { id: "initials", name: "Initials", type: "string" },
+        { id: "name", name: "Name", type: "string", canBeUsedAsSlug: true },
+        { id: "example", name: "Example", type: "boolean" },
+        {
+            id: "emails",
+            name: "Emails",
+            type: "string",
+            getValue: value => {
+                const list = v.parse(v.array(v.string()), value)
+                return list.join(", ")
+            },
+        },
+        { id: "invalid_emails", name: "Invalid Emails", type: "string" },
+        {
+            id: "phones",
+            name: "Phones",
+            type: "string",
+            getValue: value => {
+                const list = v.parse(v.array(v.string()), value)
+                return list.join(", ")
+            },
+        },
+
+        { id: "positive_ratings", name: "Ratings", type: "number" },
+        { id: "photo_thumb_url", name: "Photo Thumb Url", type: "string" },
+        { id: "has_avatar", name: "Has Avatar", type: "boolean" },
+        { id: "pending_result_request", name: "Pending Result Request", type: "boolean" },
+        { id: "tasks_count", name: "Tasks Count", type: "number" },
+        { id: "my_upcoming_event", name: "My Upcoming Event", type: "boolean" },
+        { id: "followed", name: "Followed", type: "boolean" },
+        { id: "viewed", name: "Viewed", type: "boolean" },
+        { id: "my_pending_result_request", name: "My Pending Result Request", type: "boolean" },
+        { id: "upcoming_event", name: "Upcoming Event", type: "boolean" },
+        { id: "rating_visible", name: "Rating Visible", type: "boolean" },
+        { id: "ratings_count", name: "Ratings Count", type: "number" },
+        { id: "unread_notifications", name: "Unread Notifications", type: "boolean" },
+        { id: "is_revealed", name: "Is Revealed", type: "boolean" },
+        { id: "is_hired", name: "Is Hired", type: "boolean" },
+        { id: "my_last_rating", name: "My Last Rating", type: "string" },
+        { id: "source", name: "Source", type: "string" },
+        { id: "last_message_at", name: "Last Message At", type: "string" },
+        { id: "adminapp_url", name: "Admin App Url", type: "string" },
```

Comment:
```suggestion
        { id: "adminapp_url", name: "Admin App URL", type: "link" },
```

Import as a link field so it can be used on a button or link on your website.